CONTRIBUTING.md

```diff
@@ -133,6 +133,59 @@ All commits should be tagged. Tags are denoted by square brackets (`[]`) and com
 In general almost all commits should fall into one of the above categories. In the cases where they don't please submit
 your PR untagged.
 
+#### Commit Labeling
+
+All commits should be labeled. Commit labeling for changelog and backporting is enforced in CI, but labels may only be
+applied by project maintainers. PRs from non-maintainers will be labeled by maintainers prior to a PR being accepted and merged.
+
+**Changelog Labels**
+
+Labels used for the changelog include `skip-changelog` which should be used if the PR should not be considered for the changelog,
+and any labels listed in the [root package.json's changelog config](https://github.com/emberjs/data/blob/master/package.json#L154).
+These labels are prefixed with `changelog:` and currently the options are:
+
+- `changelog:breaking` which should be used to signify a breaking change
+- `changelog:feat` which should be used to signify an addition of a new public feature or behavior
+- `changelog:bugfix` which should be used to signify a fix for a reported issue
+- `changelog:perf` which should be used to signify that the commit will improve performance characteristics in a meaningful way
+- `changelog:cleanup` which should be used to signify removal of deprecated features or that a deprecation has become an assertion.
+- `changelog:deprecation` which should be used to signify addition of a new deprecation
+- `changelog:doc` which should be used to signify a fix or improvement to documentation generated for api.emberjs.com
+- `changelog:test` which should be used to signify addition of new tests or refactoring of existing tests
+- `changelog:chore` which should be used to signify refactoring of internal code that should not have an affect on public APIs or behaviors but which we may want to call out for potentially unintended consequences.
```

Comment:
Do we want to mention anything about "multiple" labels?  Like cleanup + doc?

Reply:
I'm not sure, mostly because I don't think we've settled on a convention for whether we prefer only-1 (the most applicable) or multiple if more than one apply. Both have value and it is kinda situational.

Reply:
But we can add this guidance in the future if we start to find a convention we are sure of.